Makefile

```diff
@@ -77,6 +87,11 @@ clean:
 	@$(RM) $(SRCS_TSLINT_RULES_JS)
 
 
+$(SHELLCHECK_TARGET): $(SRCS_SHELL)
+	@$(RM) $(SHELLCHECK_TARGET)
+	@$(SHELLCHECK) $(SRCS_SHELL)
+	@$(TOUCH) $(SHELLCHECK_TARGET)
```

Comment:
I see. Since the target is fake it would be helpful to have it identified as such. example SHELLCHECK_FAKE_TARGET.